docs/modules/user-guide/partials/proc_migrating-components.adoc

```diff
@@ -2,23 +2,14 @@
 = Migrating components
 
 [role="_abstract"]
-This section describes how to migrate components to devfiles v2.x. The components section of devfiles v2.x has major changes. A component specified in a devfile v1.x no longer works in a devfile v2.x.
+This section describes how to migrate components to devfiles v2.x. Major changes have been made from devfiles v1.x to the components section in devfiles v2.x. A component specified in a devfile v1.x no longer works in a devfile v2.x.
 
 .Procedure
 
-<<<<<<< HEAD
-<<<<<<< HEAD
 . Component is a polymorphic type
 +
-For a better syntax validation component now is defined as a polymorphic type and can be implemented as `container`, `kubernetes`, `openshift`, `plugin` or `volume`.
+For a better syntax, the validation component is now defined as a polymorphic type and can be implemented as `container`, `kubernetes`, `openshift`, `plugin` or `volume`.
```

Comment:
The meaning of those two sentences is a different one, right?
"For a better syntax, the validation component is now defined as a polymorphic type and can be implemented as ..."
"To get a better syntax validation, define a component as a polymorphic type and implement it as"

What I am asking is, since it is different, is the newer version still correct?
If yes, syntax of what? Better syntax sounds a bit unspecific to me, but I'm not too familiar with the matter, of course!

Reply:
I think you are correct with the phrasing, "To get a better syntax validation, define a component as a polymorphic type and implement it as:" 

I'll make that change!

Reply:
resolving as I have addressed this concern.